### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by PlantBrush**
https://salmon-mushroom-048c5f11e.1.azurestaticapps.net/

### CODE REVIEW

**Comment by BenagainB** [comment]
I tested your link, which works for me, and the page looks fully functional. If you want, you can also add a link to the page from the menu at the top left corner, which is added in the layouts/default.vue file.

File: wordle-web/pages/index.vue [hunk 1]
```diff
@@ -89,21 +83,10 @@
...
-
-  changeButtonText() {
-    this.buttonText =
-      this.buttonText === 'Inspire me!' ? 'Inspire me again!' : 'Inspire me!'
-    this.isLoading = true
-    setTimeout(() => {
-      this.isLoading = false
-    }, 1000)
-  }
+export default class IndexPage extends Vue{
```
Comment (hewrtarkhany) [inline wordle-web/pages/index.vue:89]: i recommend more commits with small changes, but awesome job

**Comment by mmwoodfo** [comment]
In the future, just make sure that you are making a separate branch for your assignment, rather than making a PR using your main. Code changes showing up here in the PR should only be relevant to the assignment the PR is for.

Here is an example of a PR that uses an Assignment 1 branch: https://github.com/IntelliTect-Samples/EWU-CSCD379-2022-Spring/pull/11

Great job!

**Comment by mmwoodfo** [comment]
Did you have a partner for this?

**Pull request closed without merging**